Sprint 3 deployment glitch

"Module not found: Error: Can't resolve '../components/ProfileBlock' in '/tmp/build_821f0e77/src/pages' "
This is caused by heroku looking for __P__rofileBLock but only finding __p__rofileBlock. For some reason, this was never an issue with development. Must be an operating system thing

--- a/src/pages/search.js
+++ b/src/pages/search.js
@@ -3,7 +3,7 @@ import axios from 'axios';
 import Navigation from '../components/Navigation';
 import pagecss from './page.css'
 import Post from '../components/post'
-import ProfileBlock from '../components/ProfileBlock';
+import ProfileBlock from '../components/profileBlock';
 
 import MustLogin from '../components/mustLogin';
 

